Cargo.toml

```diff
@@ -126,12 +126,18 @@ opt-level = 3
 [profile.release]
 panic = "abort"
 lto = true
+# https://github.com/rust-lang/rust/issues/47745
+# https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units
+codegen-units = 1
 #strip = "symbols"      # TODO: uncomment once stable
 
 [profile.min-size-release]
 inherits = "release"
 lto = true
 opt-level = "z"
+# https://github.com/rust-lang/rust/issues/47745
+# https://doc.rust-lang.org/rustc/codegen-options/index.html#codegen-units
+codegen-units = 1
```

Comment:
That's normally not needed, since we have `inherits=release` above

Reply:
I've added the duplication upon @melekes request and I just removed it. I also removed the `lto` that was a duplication of release profile.